ingestors/npm.go

```diff
@@ -1,154 +1,116 @@
 package ingestors
 
 import (
-	"context"
 	"fmt"
-	"time"
+	"io"
+	"strconv"
 
+	"github.com/buger/jsonparser"
 	log "github.com/sirupsen/logrus"
 
-	kivik "github.com/go-kivik/kivik/v4"
-
-	_ "github.com/go-kivik/kivik/v4/couchdb"
 	"github.com/librariesio/depper/data"
 )
 
-const NPMRegistryHostname = "https://replicate.npmjs.com"
-const NPMRegistryDatabase = "registry"
+const npmSchedule = "1-59 * * * *"
+const npmIndexUrl = "https://replicate.npmjs.com/registry/_changes"
 
-// Delay between attempts at initial connection to Changes() feed
-const ConnectRetryDelaySeconds = 60 * 5
+// Arbitrary sequence number from 5/20/2025, just to make sure we
+// don't fallback to "0". Via https://replicate.npmjs.com/registry/.
+const fallbackSequence = 42540679
```

Comment:
I don't love this long term but seems fine for today (would love to fallback to figuring out a sequence number if we don't have one but not sure if there's a great way to do that)

Reply:
Yeah, that crossed my mind too but wasn't sure if it was worth the complexity. I can give it a shot.